### PR TITLE
Pipelines on developer perspective

### PR DESCRIPTION
**Pull request by Preeticp**

### CODE REVIEW

File: modules/ROOT/pages/assembly_working-with-pipelines-using-the-developer-perspective.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+You can use the *Developer* perspective of the OpenShift web console to create CI/CD pipelines for your software delivery process while creating an application on OpenShift Container Platform.
+
+NOTE: The .NET Core runtime is not yet supported by the pipeline templates.
+
+include::proc_creating-applications-with-cicd-pipelines.adoc[leveloffset=+1]
+
+include::proc_interacting-with-pipelines-using-the-developer-perspective.adoc[leveloffset=+1]
+
+:context: {parent-context-of-working-with-pipelines-using-developer-perspective}
```
Comment (nkakkar81) [inline modules/ROOT/pages/assembly_working-with-pipelines-using-the-developer-perspective.adoc:26]: Not sure why there are two contexts added here. I have limited knowledge of this, but just wanted to bring it up

File: modules/ROOT/pages/proc_interacting-with-pipelines-using-the-developer-perspective.adoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+// assembly_working-with-pipelines-using-developer-perspective.adoc
+
+[id="interacting-with-pipelines-using-the-developer-perspective_{context}"]
+= Interacting with Pipelines using the Developer perspective
+
+After you create Pipelines, you can visually interact with your deployed Pipelines on the *Developer* perspective of the OpenShift web console.
+
+The *Pipelines* view in the *Developer* perspective lists all the pipelines in a project along with details, such as the namespace in which the pipeline was created, the last pipeline run, the status of the tasks in the pipeline run, the status of the pipeline run, and the time taken for the run.
+
+To visually interact with the pipelines in your project:
```
Comment (nkakkar81) [inline modules/ROOT/pages/proc_interacting-with-pipelines-using-the-developer-perspective.adoc]: I think this sentence can be removed. The topic is self explanatory.

File: modules/ROOT/pages/proc_interacting-with-pipelines-using-the-developer-perspective.adoc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+[Discrete]
+=== Prerequisites
+Ensure that:
```
Comment (nkakkar81) [inline modules/ROOT/pages/proc_interacting-with-pipelines-using-the-developer-perspective.adoc]: I think "Ensure that:" can also be removed.

File: modules/ROOT/pages/proc_interacting-with-pipelines-using-the-developer-perspective.adoc [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+* You have access to an OpenShift Cluster and have logged in to the web console and are in the *Developer* perspective.
```
Comment (nkakkar81) [inline modules/ROOT/pages/proc_interacting-with-pipelines-using-the-developer-perspective.adoc]: too many 'and's. Do think we can break the sentence into two?

**Review by nkakkar81 — APPROVED** [review]

File: modules/ROOT/pages/proc_interacting-with-pipelines-using-the-developer-perspective.adoc [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
++
+. Click on the required pipeline to see the *Pipeline Details* page. It provides a visual representation of all the serial and parallel tasks in the pipeline. The tasks are also listed to the bottom-right of the page. You can click the *Tasks* listed to view the details of the task.
++
+image::pipeline_details.png[Pipeline details]
++
+. In the *Pipeline Details* page :
+
+* Click *Pipeline Runs* tab to see the completed, running, or failed runs for the pipeline. You can use the Options menu (icon image)  to—stop a running pipeline, rerun a pipeline using the same parameters and resources as that of the previous pipeline execution, or delete a pipeline run.
+* Click *Parameters* tab to see the parameters defined in the Pipeline. You can also add or edit additional parameters as required.
+*Click *Resources* tab to see the resources defined in the Pipeline. You can also add or edit additional resources as required.
```
Comment (nkakkar81) [inline modules/ROOT/pages/proc_interacting-with-pipelines-using-the-developer-perspective.adoc]: Add space between * and Click

File: modules/ROOT/pages/proc_interacting-with-pipelines-using-the-developer-perspective.adoc [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+. Click *Actions* on the upper right of the screen and select *Start* to start the pipeline. The *Start Pipeline* dialog box displays the *Git Resources*, and *Image Resources* based on the pipeline definition.
```
Comment (nkakkar81) [inline modules/ROOT/pages/proc_interacting-with-pipelines-using-the-developer-perspective.adoc]: *the* Image Resources

File: modules/ROOT/pages/assembly_working-with-pipelines-using-the-developer-perspective.adoc (hunk 1 above)
Comment (Preeticp) [reply modules/ROOT/pages/assembly_working-with-pipelines-using-the-developer-perspective.adoc:26]: Based on mod docs template, let's discuss in detail later.

**Pull request merged**